### PR TITLE
Support Packer 1.5.0

### PR DESCRIPTION
**Pull request by tomelliff**
As mentioned in https://www.packer.io/guides/hcl/component-object-spec/

Needed to move to Go modules to get HCL2 support working as `dep` wasn't happy with it.

### CODE REVIEW

File: post-processor.go [hunk 1]
```diff
@@ -1,9 +1,13 @@
+//go:generate mapstructure-to-hcl2 -type Config
+
 package main
 
 import (
+	"context"
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/hashicorp/hcl/v2/hcldec"
```
Comment (tomelliff) [inline post-processor.go]: Whoops, this needs putting in another block. I'll fix when I get a sec.

**Comment by martinbaillie** [comment]
Looking great @tomelliff — I might ditch `vendor` after merge with Go modules now. Is it g2g from your perspective?

**Comment by tomelliff** [comment]
Yep, it's working for me with the nightly Packer release or building from source.

Still errors with Packer 1.5.1 but this at least unblocks being able to use this with Kubernetes service accounts based IAM roles for our CI pipeline for now as our CI runner is building both Packer and this post processor from source right now.

I added the `-mod=vendor` flag to the build step in the Makefile so if you ditch the vendor folder then that also needs to go.

**Comment by martinbaillie** [comment]
@tomelliff this is available as v1.0.8

**Pull request merged**